classifier.go

```diff
@@ -10,7 +10,7 @@ import (
 // The scores must order the candidates so as the highest score be the most probably language of the content. The candidates is
 // a map which can be used to assign weights to languages dynamically.
 type Classifier interface {
-	Classify(content []byte, candidates map[string]float64) map[string]float64
+	Classify(content []byte, candidates map[string]float64) (scores map[float64]string)
```

Comment:
I don't think it's much harder to do it properly (even if you have to write a comparator function), and it should certainly be more efficient. OTOH, using a map will break as soon as you have 2 languages with the same weight.